src/worker.cpp

```diff
@@ -404,7 +404,7 @@ void Worker::setDeviceTreeAndJson()
 {
     types::VPDMapVariant parsedVpdMap;
     fillVPDMap(SYSTEM_VPD_FILE_PATH, parsedVpdMap);
-    
+
```

Comment:
Remove this.

Reply:
Remove this means? You want me to skip this file formatting changes as part of this commit? Please confirm .

Reply:
This is just an extra line added. Shouldn't be because of clang formatting.

Reply:
no, not added extra line. In older code there was a space/tab in line 407, that has been removed in newer code. that's the change and it was done by formatting script. 
But now I have removed those files which had just formatting changes. so should be ok now.